src/documentation/examples/SimpleModal/SimpleModal.jsx

```diff
@@ -7,19 +7,31 @@ import Button from '@bufferapp/ui/Button';
 export default function ExampleSimpleModal() {
   const [modalOpen, openModal] = useState(false);
   const [changed, setChanged] = useState(false);
+  const [showingModal, setShowingModal] = useState(false);
+
   return (
     <div>
       <Button
+        disabled={showingModal}
         type="primary"
         onClick={() => {
           openModal(true);
+          setTimeout(() => {
+            setShowingModal(true)
+          }, 200)
```

Comment:
This is used to test the issue, as it's causing the parent to re-render during the staging in animation.